extra/modules/ortb2-blocking/src/main/java/org/prebid/server/hooks/modules/ortb2/blocking/core/model/BlockedAttributes.java

```diff
@@ -0,0 +1,22 @@
+package org.prebid.server.hooks.modules.ortb2.blocking.core.model;
+
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.List;
+import java.util.Map;
+
+@Builder
+@Value
+public class BlockedAttributes {
+
+    List<String> badv;
```

Comment:
I'm really in favor of commenting all this code with at least:

- what the acronym stands for. Even after 4 years in ad tech I forget about half of these and they can be ambiguous quite often too
- a link to where this attribute has it's origin if it has one. A lot of them have a section in the OpenRTB Spec. Referencing it would be pretty good. 

I'm mentioning this, because one can get lost in the code base quite fast. Even with domain - knowledge and a bit of Java experience. Commenting the acronyms and the "why does this code exist" is a tremendous help for souls like me 😁